python/ray/_raylet.pyx

```diff
@@ -946,7 +955,7 @@ cdef class CoreWorker:
         with self.profile_event(b"submit_task"):
             if num_method_cpus > 0:
                 c_resources[b"CPU"] = num_method_cpus
-            task_options = CTaskOptions(num_return_vals, c_resources)
+            task_options = CTaskOptions(num_return_vals, False, c_resources)
```

Comment:
This should be an argument to `SubmitTask` given that it only applies to normal tasks

Reply:
Uh the value doesn't really matter for actor tasks since it's fixed at actor creation.